clap_derive/src/utils/doc_comments.rs

```diff
@@ -83,14 +83,64 @@ fn split_paragraphs(lines: &[&str]) -> Vec<String> {
         last_line += start + len;
 
         if len != 0 {
-            Some(merge_lines(&slice[..len]))
+            Some(process_paragraph(&slice[..len]))
         } else {
             None
         }
     })
     .collect()
 }
 
+fn process_paragraph(lines: &[&str]) -> String {
+    lines
+        .iter()
+        .map(|ln| ln.trim())
```

Comment:
Maybe I'm misreading the [commonmark spec](https://spec.commonmark.org/0.30/#hard-line-break) but it doesn't sound like there can be spaces between `\\` and `\n` which a trim would allow.

Reply:
Just checked the commonmark preview.  one space after leaves a `\\`.  More then one space after is treated as `\\\n` because of another hard break rule.